build/Build.Pack.cs

```diff
@@ -257,11 +259,26 @@ void BuildMsiInstallerForPlatform(MSBuildTargetPlatform platform, AbsolutePath w
                 .SetOutputDirectory(ArtifactsDirectory / "chocolatey"));
         });
 
+    [PublicAPI]
+    Target PackContracts => _ => _
+        .Description("Packs the NuGet package for Tentacle contracts.")
+        .Executes(() =>
+        {
+            DotNetPack(p => p
+                .SetProject(RootDirectory / "source/Octopus.Tentacle.Contracts/Octopus.Tentacle.Contracts.csproj")
+                .SetVersion(OctoVersionInfo.FullSemVer)
+                .SetOutputDirectory(ArtifactsDirectory)
+                .DisableIncludeSymbols()
+                .SetVerbosity(DotNetVerbosity.Normal)
+                .SetProcessArgumentConfigurator(_ => _ .Add("/p:NoWarn=NU5104")));
+        });
+
     [PublicAPI]
     Target PackWindows => _ => _
         .Description("Packs all the Windows targets.")
         .DependsOn(PackWindowsZips)
-        .DependsOn(PackChocolateyPackage);
+        .DependsOn(PackChocolateyPackage)
+        .DependsOn(PackContracts);
```

Comment:
Only on Windows?

Reply:
It's a cross-platform package, so only needs to be packed once. I'm piggybacking on the Windows build, but could be part of any of the builds or have it's own. What do you think would be the cleanest/clearest approach?

Reply:
I think having it as a dependency on `Pack` would be the clearest. On TeamCity we never call `Pack` though. We directly call `PackCrossPlatformBundle`. So we could either:

- Call `Pack` on TeamCity instead of `PackCrossPlatformBundle` and update the name of the build step
- Add another build step for calling `PackContracts`

Not sure which would be better, both sound ok to me.

Reply:
I like the idea of decoupling it from the Window pack. I think a build step that runs `PackContracts` is a nice solution.